### PR TITLE
docs: add 2.0 change log and mention redis in upgrade instructions

### PR DESCRIPTION
**Pull request by alexmt**
Signed-off-by: Alexander Matyushentsev <AMatyushentsev@gmail.com>

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,109 @@
...
+
+### Banner Feature
+
+Want to notify your Argo CD users of upcoming changes? Just specify the notification message and optional URL using the
+`ui.bannercontent` and `ui.bannerurl` attributes in the `argocd-cm` ConfigMap. 
+
+### Core Features
+
+* The new sync option `PrunePropagationPolicy=background` allows using foreground deletion during syncing
+* New application finalizer `resources-finalizer.argocd.argoproj.io:background` allows using foreground deletion when the application is deleted
```
Comment (jannfis) [inline CHANGELOG.md:31]: I guess you mean background in both sentences :)

File: docs/operator-manual/upgrading/1.8-2.0.md [hunk 2]
```diff
@@ -1,5 +1,13 @@
 # v1.8 to v2.0
 
+## Redis Upgraded to v6.2.1
+
+The bundled Redis version has been upgraded to v6.2.1.
+
+The Redis itself should be able to upgrade with no downtime, as well as Argo CD does use it as a persistent store.
```
Comment (jannfis) [inline docs/operator-manual/upgrading/1.8-2.0.md:7]: Do you mean "does not use it as persistent store" here?

**Review by jannfis — COMMENTED** [review]
Found some things you should take a look at. Also, maybe we should link from the changelog to the upgrade docs?

File: CHANGELOG.md (hunk 1 above)
Comment (alexmt) [reply CHANGELOG.md:31]: thank you! fixed it here and in release blog as well :)

File: docs/operator-manual/upgrading/1.8-2.0.md (hunk 2 above)
Comment (alexmt) [reply docs/operator-manual/upgrading/1.8-2.0.md:7]: good catch. Thank you!

**Comment by alexmt** [comment]
Thank you for review @jannfis . Fixed bugs and added upgrade instructions link

**Review by jannfis — APPROVED** [review]
LGTM

**Pull request merged**